inferno-framework/inferno-core
# Fi 3597 fix js build

# Summary
This branch removes an outdated js file and updates the publish script to automatically remove old js files prior to performing a new js build.

# Testing Guidance

* Comment out the `gem push` line of `publish.sh`.
* `./publish.sh`

In SMART test kit:

* `mkdir inferno`
* copy `inferno_core-0.5.3.gem` from inferno core into the inferno directory
* `cd inferno`
* `tar -xvf inferno_core-0.5.3.gem`
* `tar -xvzf data.tar.gz`
* copy `inferno_core.gemspec` from inferno core into the inferno directory
* `cd ..`
* comment out the inferno core requirement in `smart_app_launch_test_kit.gemspec`
* add `gem 'inferno_core', path: './inferno'` to the `Gemfile`
* `bundle`
* run the test kit
* verify that the auth info choices are working as expected
